src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -112,6 +112,12 @@ abstract class AbstractNormalizer implements NormalizerInterface, DenormalizerIn
      */
     public const IGNORED_ATTRIBUTES = 'ignored_attributes';
 
+    /**
+     * The default group can be use on the serialization context to explicitly
+     * tell the serializer to allow properties without defined groups.
+     */
+    public const DEFAULT_GROUP_FOR_ATTRIBUTE_WITHOUT_GROUPS = '_default';
```

Comment:
Is this supposed to be overridden in the userland? 
If not, we could mark it as final I guess, and maybe move it on top of L36 so that it's not considered as a context constant.
If yes, then why not store it in the context (and update the `AbstractNormalizerContextBuilder`)?